Assets/LeapMotionModules/ElementRenderer/Scripts/Spaces/LeapGuiCylindricalSpace.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[AddComponentMenu("")]
+[LeapGuiTag("Cylindrical")]
+public class LeapGuiCylindricalSpace : LeapGuiRadialSpace, ISupportsAddRemove {
+  public const string FEATURE_NAME = LeapGui.FEATURE_PREFIX + "CYLINDRICAL";
+
+  private Dictionary<Transform, Transformer> _transformerData = new Dictionary<Transform, Transformer>();
+
+  public void OnAddElements(List<LeapGuiElement> element, List<int> indexes) {
+    BuildElementData(transform); //TODO, optimize
+  }
+
+  public void OnRemoveElements(List<int> toRemove) {
+    BuildElementData(transform); //TODO, optimize
+  }
+
+  public override void BuildElementData(Transform root) {
+    _transformerData.Clear();
+
+    refreshElementData(root, new Transformer() {
+      space = this,
+      anchor = root,
+      angleOffset = 0,
+      heightOffset = 0,
+      radiusOffset = radius,
+      radiansPerMeter = 1.0f / radius
+    });
+  }
+
+  public override void RefreshElementData(Transform root) {
+    //TODO!
+    //refreshElementData(root, root);
```

Comment:
I should have made a note, this method is a stub for a more efficient means of refreshing element space data, but there is no logic in LeapGui for handling this, so it just calls BuildElementData every frame to rebuild everything.  In another PR I will add the more efficient means of updating element data.